.github/workflows/build-dependencies.txt

```diff
@@ -33,12 +36,15 @@ mdadm
 nfs-kernel-server
 pamtester
 parted
+po-debconf
 python3
+python3-all-dev
 python3-cffi
 python3-dev
 python3-packaging
 python3-pip
 python3-setuptools
+python3-sphinx
```

Comment:
Why do we need this package? IIRC we don't have anything in main repository to use Sphinx on

Reply:
This is a dependency mentioned in `contrib/debian/control`. In `contrib/debian/rules.in`, sphinx docs can be generated. I think that requires this package.